Sources/_InternalTestSupport/misc.swift

```diff
@@ -193,7 +193,6 @@ public enum TestError: Error {
     case platformNotSupported
 }
 
-@available(*, deprecated, message: "Migrate test to Swift Testing and use 'fixture' instead")
```

Comment:
issue: Please reading this as we want to to encourage contributor to migrate away from XCTest when they touch a test file.